src/pokemon/Pokemon.tsx

```diff
@@ -45,17 +50,18 @@ export function Pokemon({ name, order, weight, height, types, sprites }: PokeApi
                     <div className={styles.pokemonViewerScreen}>
                         <img className={styles.pokemonImage} src={spriteUrl} alt={name} />
                     </div>
-                    <div className={styles.pokemonButtonContainer} style={{ justifyContent: hasFemale ? 'space-between' : 'flex-end' }}>
+                    <div className={styles.pokemonButtonContainer}>
                         {
                             hasFemale ?
                                 <div className={styles.genderButtonImages}>
-                                    <ButtonImage name='default' size='button' color={gender === 'default' ? 'black' : '#CACACA'} onClick={onClick} disabled={gender === 'default'} />
-                                    <ButtonImage name='female' size='button' color={gender === 'female' ? 'black' : '#CACACA'} onClick={onClick} disabled={gender === 'female'} />
+                                    <ButtonImage name='default' size='button' color={gender === 'default' ? colorButtonActive : colorButtonDisabled} onClick={onClick} disabled={gender === 'default'} />
+                                    <ButtonImage name='female' size='button' color={gender === 'female' ? colorButtonActive : colorButtonDisabled} onClick={onClick} disabled={gender === 'female'} />
                                 </div>
-                                : null
+                                : <div className={styles.emptyDiv}></div>
                         }
+                        <ButtonImage name={bookmark} size='button' onClick={() => setBookmark(() => bookmark === 'bookmark-on' ? 'bookmark-off' : 'bookmark-on')} />
                         <div className={styles.shinyOrientationContainer}>
-                            <ButtonImage name='shiny' size='button' color={shiny ? 'black' : '#CACACA'} onClick={() => setShiny(!shiny)} />
+                            <ButtonImage name='shiny' size='button' color={shiny ? colorButtonActive : colorButtonDisabled} onClick={() => setShiny(!shiny)} />
```

Comment:
Tu as trois fois la même ternaire, ça peut se refacto en fonction ! 
`const getColor = (v: boolean) => v ? colorButtonActive : colorButtonDisabled`

Reply:
C'est noté ! pour simplifier le process je merge cette PR mais je m'en souviendrais pour une prochaine fois